requirements.txt

```diff
@@ -10,7 +10,7 @@ Flask-Bootstrap==3.3.5.7
 
 # External libraries
 SQLAlchemy==1.0.11
-twilio==6.0.0rc12
+twilio>=6.0.0rc13
```

Comment:
This is pulling `6.3.dev0`, lock it to `6.0.0` instead or you can try using the `~` operator: 
https://github.com/pypa/pip/issues/1551